plugin/adsk/plugin/exportTranslator.cpp

```diff
@@ -57,6 +61,8 @@ UsdMayaExportTranslator::writer(const MFileObject &file,
     double frameStride = 1.0;
     bool append=false;
     
+    std::set<double> frameSamples;
```

Comment:
A default/empty set<double> was always passed in down below, now parsing the input options to see if the user set them in the UI.